src/category_theory/cofiltered_system.lean

```diff
@@ -48,13 +52,79 @@ Mittag-Leffler, surjective, eventual range, inverse system,
 
 -/
 
-universes u v
+universes u v w
+
+open category_theory category_theory.is_cofiltered set category_theory.functor_to_types
+
+section fintype_konig
+
+/-- This bootstraps `nonempty_sections_of_fintype_inverse_system`. In this version,
+the `F` functor is between categories of the same universe, and it is an easy
+corollary to `Top.nonempty_limit_cone_of_compact_t2_inverse_system`. -/
+lemma nonempty_sections_of_fintype_cofiltered_system.init
+  {J : Type u} [small_category J] [is_cofiltered J] (F : J ⥤ Type u)
+  [hf : Π (j : J), fintype (F.obj j)] [hne : Π (j : J), nonempty (F.obj j)] :
```

Comment:
I'd take this opportunity to replaced `fintype` by `finite` and `is_cofiltered` by `is_cofiltered_or_empty`.

Reply:
Ah, OK, so the linter wanting `finite` instead of `fintype` _should_ also apply to `nonempty_sections_of_…` ?